docker.Makefile

```diff
@@ -37,7 +37,8 @@ cross: build_docker_image
 
 # start container in interactive mode for in-container development
 dev: build_docker_image
-	@docker run -ti $(MOUNTS)  $(DEV_DOCKER_IMAGE_NAME) ash
+	@docker run -ti $(MOUNTS) -v /var/run/docker.sock:/var/run/docker.sock $(DEV_DOCKER_IMAGE_NAME) ash
```

Comment:
I am sorry but I don't understand why mounting the docker socket is necessary for `cross`.
`cross` already has `$(MOUNTS)` though, in case you are thinking about mounting the sources :)

I can add it now if necessary.